### ISSUE TITLE
Wrong error alert for Chinese string

### ISSUE THREAD

**Issue by ghost** [hoovercj/vscode-ruby-linter#2]
I got an `unexpected end-of-input, expecting keyword_end` error with the code below

```ruby
if true
  puts "123你好"
end
```

But it's totally OK with `ruby -wc`
